packages/data/src/components/async-mode-provider/context.js

```diff
@@ -9,4 +9,11 @@ const { Consumer, Provider } = Context;
 
 export const AsyncModeConsumer = Consumer;
 
+/**
```

Comment:
We need an example here. It's a very complex use case, so I would also include more details. We should also share some recommendations.

Reply:
To be fair, this needs a blog post to be explained :) so I'm not really sure what to add here :)

Reply:
I'll try to expand a bit, but not sure how good it will be :P

Reply:
Documenting is hard :) You can always post on make.wordpress.org/core and reference it from here 🤷‍♂

Reply:
Updated, let me know what you think.